api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -513,29 +517,48 @@ private FirecloudWorkspaceId getFcBillingProject(DbAccessTier accessTier, Worksp
     }
     final ImmutableMap<String, WorkspaceAccessLevel> aclsByEmail = shareRolesMapBuilder.build();
 
+    List<String> usersHavePermission = new ArrayList<>();
```

Comment:
I find this name confusing, particularly since it's a top level variable in a `shareWorkspace` method (this is not scoped to workflows only). maybe `workflowUsers`?